### ISSUE TITLE
TextBillboardRenderer Only Shows Blocks

### ISSUE THREAD

**Issue by RyanMcilnay** [jzy3d/jzy3d-api#65]
The call to glBitmap only uses nonascii static byte[] instead of using the nonascii2 byte[] param, causing only blank boxes to show up.  Is this a mistake?

```
private void glBitmap(GL gl, int charWidth2, int charHeight2, float xorig, float yorig, float xmove, float ymove, byte[] nonascii2, int i) {
    if (gl.isGL2()) {
        gl.getGL2().glBitmap(charWidth, charHeight, xorig, yorig, xmove, ymove, nonascii, 0);
    } else {
        GLES2CompatUtils.glBitmap(charWidth, charHeight, xorig, yorig, xmove, ymove, nonascii, 0);
    }

}
```

**Comment by martinpernollet** [comment]
This is a bug. Thank you

2016-06-30 17:37 GMT+02:00 RyanMcilnay <notifications@github.com>:

**Comment by jzy3d** [comment]
Fixed in 2.0
